packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -2851,398 +3013,534 @@ describe.each([
             one: "foo2",
             two: "bar2",
           })
+          const three = await config.api.row.save(table._id!, {
+            one: "foo3",
+            two: "bar3",
+          })
 
           const view = await config.api.viewV2.create({
             tableId: table._id!,
             name: generator.guid(),
-            query: [
-              {
-                operator: BasicOperator.EQUAL,
-                field: "two",
-                value: "bar2",
-              },
-            ],
+            queryUI: {
+              onEmptyFilter: EmptyFilterOption.RETURN_ALL,
+              logicalOperator: UILogicalOperator.ALL,
+              groups: [
+                {
+                  logicalOperator: UILogicalOperator.ALL,
+                  filters: [
+                    {
+                      operator: BasicOperator.NOT_EQUAL,
+                      field: "one",
+                      value: "foo2",
+                    },
+                  ],
+                },
+              ],
+            },
             schema: {
               id: { visible: true },
-              one: { visible: false },
+              one: { visible: true },
```

Comment:
What is the change that lead to this? From what I can see the old test case specified some fields visible and some not, while this no longer does that, is there another test case that replicate the behaviour this was testing now?

Reply:
Weird, it hasn't changed as far as I can tell:
https://github.com/Budibase/budibase/blob/44fbe7074050a3420025638f0542fcd267bd5d78/packages/server/src/api/routes/tests/viewV2.spec.ts#L2687-L2691